src/test.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 The Foundry Visionmongers Ltd
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+
+try:
+    import openassetio
+    from openassetio.hostApi import ManagerFactory
+except ImportError as exc:
+    print("Error importing openassetio package: {}".format(exc))
```

Comment:
:D This looks suspiciously formatted like some of our existing tests ;) :D

Q: Is OpenAssetIO supported on Python 2 at all?

I know it probably shouldn't be with modern development, but, some studios are still using it.

`print` is what I'm thinking about for Py2.

Reply:
Thankfully, no Python 2 for us!

Happy to improve the formatting if you have a suggestion?

Reply:
Formatting and structure is fine IMO (biased, as I created the first instance!)

It was merely the use of print that made me ask the question about Py2.